filemage/README.md

```diff
@@ -0,0 +1,83 @@
+# Agent Check: filemage
+
+## Overview
+
+This check monitors [FileMage][1].
+
+## Setup
+
+### Installing the package
+
```

Comment:
Missed this paragraph, otherwise looks good:

```suggestion

For the Datadog Agent v7.21 or v6.21 and later, follow these instructions to install the Filemage integration on your host. See [Use Community Integrations][2] to install it with the Docker Agent or earlier versions of the Datadog Agent.

```